### PR TITLE
docs: Update docs

### PR DESCRIPTION
**Pull request by jrconlin**
Fixes typos, spilling, and other minor errors.

### CODE REVIEW

File: docs/apns.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+The APNS post message contains JSON formatted data similar to the following:
+
+.. code-block:: json
+
+    {
+        "aps": {
+            "alert": "notification title",
+            "content_available": 1
+        }
```
Comment (bbangert) [inline docs/apns.rst]: Need a comma here.

File: docs/apns.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Sending the APNS message
+------------------------
+
+The APNS post message contains JSON formatted data similar to the following:
+
+.. code-block:: json
+
+    {
+        "aps": {
```
Comment (bbangert) [inline docs/apns.rst]: Why aps instead of apns?

File: docs/index.rst [hunk 3]
```diff
@@ -22,7 +22,7 @@ the `Mozilla Push Service architecture diagram`_. This push service uses
 websockets to talk to Firefox, with a Push endpoint that implements the
 :term:`WebPush` standard for its :ref:`http` API.
 
-Autopush API's
+Autopush APIs
 ==============
```
Comment (bbangert) [inline docs/index.rst]: Length usually needs to match the text above.

File: docs/index.rst [hunk 4]
```diff
@@ -135,7 +135,7 @@ Reference
 License
 =======
 
-``autopush`` is offered under the Apache License 2.0.
+``autopush`` is offered under the Mozilla Public License 2.0.
```
Comment (bbangert) [inline docs/index.rst]: Well that's weird, wonder why that was there.

File: docs/apns.rst (hunk 2 above)
Comment (jrconlin) [inline docs/apns.rst]: Because Apple. `aps` is a reserved name for the dictionary of meta values.

File: docs/apns.rst [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        },
+        "key": "value",
+        ...
+    }
+
+
+*aps* is reserved as a sub-dictionary. All other *key*: *value* slots are open.
+
+In addition, you must specify the following headers:
+* *apns-id*: A lowercase, dash formatted UUID for this message.
```
Comment (pjenvey) [inline docs/apns.rst]: Missed an extra newline here to make this section render nicely

File: docs/apns.rst [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+* *apns-priority*: Either **10** for Immediate delivery or **5** for delayable delivery.
+* *apns-topic*: The bundle ID for the recipient application. This must match the bundle ID of
+the AppID used to create the *"Apple Push Services:..."* certificate. It usually has the format
```
Comment (pjenvey) [inline docs/apns.rst]: a few of these hanging lines need indents to render nicely too

File: docs/apns.rst [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+of `com.example.ApplicationName`.
+* *apns-expiration*: The timestamp for when this message should expire in UTC based seconds.
+"0" means immediate expiration.
+
+Handling APNS responses
+-----------------------
+
+APNS returns a status code and an optional JSON block describing the error. A list of `these
+responses are provided in the APNS documentation <https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/APNsProviderAPI.html#//apple_ref/doc/uid/TP40008194-CH101-SW18>`_
```
Comment (pjenvey) [inline docs/apns.rst]: this link seems dead

**Pull request merged**